### PR TITLE
DM-1336 New connector operations

### PR DESCRIPTION
**Pull request by sag-eweingaertner**
User docs for LWM2M registration invalidation.

### CODE REVIEW

**Review by Oezge-Akin — APPROVED** [review]

File: content/protocol-integration/lwm2m-bundle/connector-device.md [hunk 1]
```diff
@@ -40,3 +40,28 @@ Additionally, the operation can be executed regardless of the existence of the d
...
+
+### Invalidate Registrations by Endpoint
+
+This command removes the LWM2M registrations using an endpoint Id.
+
+Syntax:   `invalidateRegistrationsForEndpoint <endpointId>`
+
+Example usage:  `invalidateRegistrationsForEndpoint urn:imei:012345678901234`
+
+This command invalidates all known LWM2M registrations for the endpoint urn:imei:012345678901234.
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/connector-device.md:57]:
```suggestion
This command invalidates all known LWM2M registrations for the endpoint `urn:imei:012345678901234`.
```

File: content/protocol-integration/lwm2m-bundle/connector-device.md [hunk 2]
```diff
@@ -40,3 +40,28 @@ Additionally, the operation can be executed regardless of the existence of the d
...
+
+### Invalidate Registrations by LWM2M registration id
+
+Alternatively a LWM2M registration may be invalidated using its ID, using the following command:
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/connector-device.md:61]:
```suggestion
Alternatively an LWM2M registration may be invalidated using its ID, using the following command:
```

File: content/protocol-integration/lwm2m-bundle/connector-device.md [hunk 3]
```diff
@@ -40,3 +40,28 @@ Additionally, the operation can be executed regardless of the existence of the d
...
+
+This command removes the LWM2M registrations using an endpoint Id.
+
+Syntax:   `invalidateRegistrationsForEndpoint <endpointId>`
+
+Example usage:  `invalidateRegistrationsForEndpoint urn:imei:012345678901234`
+
+This command invalidates all known LWM2M registrations for the endpoint urn:imei:012345678901234.
+
+### Invalidate Registrations by LWM2M registration id
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/connector-device.md:59]:
```suggestion
### Invalidate registrations by LWM2M registration ID
```

File: content/protocol-integration/lwm2m-bundle/connector-device.md [hunk 4]
```diff
@@ -40,3 +40,28 @@ Additionally, the operation can be executed regardless of the existence of the d
...
+
+Alternatively a LWM2M registration may be invalidated using its ID, using the following command:
+
+Syntax:   `invalidateRegistrationById <registrationId>`
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/connector-device.md:63]:
```suggestion
Syntax: `invalidateRegistrationById <registration_ID>`
```

File: content/protocol-integration/lwm2m-bundle/connector-device.md [hunk 5]
```diff
@@ -40,3 +40,28 @@ Additionally, the operation can be executed regardless of the existence of the d
...
+
+Example usage:  `invalidateRegistrationById F7DqjmW3Yy`
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/connector-device.md:65]:
```suggestion
Example usage: `invalidateRegistrationById F7DqjmW3Yy`
```

File: content/protocol-integration/lwm2m-bundle/connector-device.md [hunk 6]
```diff
@@ -40,3 +40,28 @@ Additionally, the operation can be executed regardless of the existence of the d
...
+
+This command invalidates the LWM2M registration with the ID F7DqjmW3Yy.
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/connector-device.md:67]:
```suggestion
This command invalidates the LWM2M registration with ID `F7DqjmW3Yy`.
```

File: content/protocol-integration/lwm2m-bundle/connector-device.md [hunk 7]
```diff
@@ -40,3 +40,28 @@ Additionally, the operation can be executed regardless of the existence of the d
 Therefore, it is also possible to use it to remove the link between the device and the tenant but still keep the device's history.
 
 Example usage: `cleanupClusterTenantObjectsForEndpoint urn:imei:012345678901234`
+
+<a name="lwm2m-invalidate-lwm2m-registrations"></a>
+## Invalidate Registrations
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/connector-device.md:45]:
```suggestion
## Invalidate registrations
```

**Comment by rssag** [comment]
which releases does this apply to?

**Comment by sag-eweingaertner** [comment]
> which releases does this apply to? @sag-eweingaertner

10.16+

**Review by rssag — APPROVED** [review]

**Pull request merged**